spec/configuration_loader/remote_v1_spec.lua

```diff
@@ -1,29 +1,68 @@
-local loader = require 'apicast.configuration_loader.remote_v1'
+local _M = require 'apicast.configuration_loader.remote_v1'
```

Comment:
This commit is not related to the rest of the PR, so it should be in a different one.

Reply:
Changed.